.env.example

```diff
@@ -1,6 +1,6 @@
 APP_ENV=local
 APP_DEBUG=true
-APP_KEY=SomeRandomKey!!!
+APP_KEY=SetARandomEncryptionKeyStringPlz
```

Comment:
Haha, was getting an error because the default key was too short and throws an error when using this for unit tests! :stuck_out_tongue: